src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PlacementRequestEntity.kt

```diff
@@ -114,6 +114,7 @@ interface PlacementRequestRepository : JpaRepository<PlacementRequestEntity, UUI
         )
       )
       AND ((CAST(:apAreaId AS pg_catalog.uuid) IS NULL) OR area.id = :apAreaId)
+      AND ((CAST(:cruManagementAreaId AS pg_catalog.uuid) IS NULL) OR apa.cas1_cru_management_area_id = :cruManagementAreaId)
```

Comment:
it just occurred to me that we should add an index on approved_premises_applications cas1_cru_management_area_id. Can maybe do that in the next PR (which will be filtering on the same value)

Reply:
Haven't done this in this PR - wanted to get this one over the line.